decisions/0006-program-slugs-in-publisher.rst

```diff
@@ -0,0 +1,30 @@
+6. Program Slugs in Publisher
+-------------------------------
+
+Status
+------
+
+Accepted (July 2023)
+
+Context
+-------
+
+Ability to update the slugs for programs is needed to be accessible to a group of users.
+These slugs are used in generating marketing url for each type of programs.
+These program slugs will be programmatically generated, just like courses.
+In order to grant the ability to update slugs for programs, we will going to display programs on Publisher due to it's ease of access control.
```

Comment:
due to its ease of access (access control is on backend, so I cant say for sure if access control is the correct word here).